src/main/java/stream/flarebot/flarebot/commands/moderation/SetPrefixCommand.java

```diff
@@ -28,7 +28,6 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         .queue();
             } else {
                 MessageUtils.sendErrorMessage("Cannot set the prefix to be more that one character long!", channel, sender);
-                return;
```

Comment:
Why was this removed?

Reply:
Prefix has no usage message, everything is handled inside the command.